src/apps/chatter/reducers/room-passwords.js

```diff
@@ -0,0 +1,15 @@
+export const roomPasswords = (action, state) => {
```

Comment:
Is the purpose so users don't have to enter password again if they navigate between different rooms?

Reply:
Yes, that's right